README.md

```diff
@@ -1 +1,19 @@
-# TestRepo
\ No newline at end of file
+# Selenium first steps
+
+Script(Java language) for Selenium WebDriver that opens Facebook.com starting page.
```

Comment:
1. You don't need to be that specific that it is Java **language**. You can just say `Java` without `language`
2. You need a space after `Script` and `(Java ...`  You need to have spaces before `(`